### ISSUE TITLE
core.setOutput not being recognized as property

### ISSUE THREAD

**Issue by estokes-vs** [rhysd/actionlint#104]
When using the following @actions/core package, I'm setting an output and then trying to reference it in a downstream action.  The output is available and proper, but `actionlint` is throwing the following error which indicates that the only property of `outputs` that is available is `results`:

`.github/workflows/ci.yml:21:33: property "color" is not defined in object type {result: string} [expression]`

When calling `${{ steps.assign.outputs.results }}` it doesn't return anything.  I'm not sure if I'm doing something wrong or frowned upon with this mechanism, but wanted to open this issue to see if anyone else has ran into the problem?

[Playground](https://rhysd.github.io/actionlint/#eJx9jrEKAjEQRPv7iimEqxKs8yticTkWE5FsyO4qcty/m2jtVVO8N8xwCagmabpzlDABSqIjgWZFHHdu0YqaeyyDfZEoVflZgBtmAK2JMZ+2DVSel/MV+z7/M/qYX7nokgs1z3Lk3rImi75RZcnK7e351VtHlX5g4A+nCkLw) (which I think is awesome)

Code excerpt of how the setOutput command functions:
https://github.com/actions/toolkit/blob/27f76dfe1afb2b7e5e679cd8e97192d34d8320e6/packages/core/src/core.ts#L190
```
export function setOutput(name: string, value: any): void {
  process.stdout.write(os.EOL)
  issueCommand('set-output', {name}, value)
}
```

Github Action example:
```
jobs:
  test-workflow:
    name: color is cool
    runs-on: ubuntu-latest
    steps:
      - id: assign
        uses: actions/github-script@v5
        with:
          script: |
            core.setOutput('color', 'pink');
      - id: call
        run: |
          echo "color is ${{ steps.assign.outputs.color }}"
```

Screenshot of output being available and read successfully:
![image](https://user-images.githubusercontent.com/80277826/148616236-2c5eea1c-e5e3-4db3-a030-b9aa10935336.png)

**Comment by rhysd** [comment]
This use case was not assumed because [the readme doc](https://github.com/actions/github-script#readme) describes the usage with `outputs.result` only.

**Comment by rhysd** [comment]
This was fixed at 1.6.11.

**Comment by TomerFi** [comment]
Just sharing:

The current version the [super-linter](https://github.com/github/super-linter) is [4.9.2](https://github.com/github/super-linter/releases/tag/v4.9.2),
still includes [version 1.6.10 of actionlint](https://github.com/github/super-linter/blob/v4.9.2/Dockerfile#L24).

I [switched to using latest](https://github.com/TomerFi/version-bumper/commit/253efee62f55538b39fbdf62439307aac05b14b1) to work around this until a new version of _super-linter_ will be released,
as it seems the fix was [already staged](https://github.com/github/super-linter/blob/20700fdb5559ae2dd11fbe378516c1eda613072b/Dockerfile#L24), but not yet released.
